### ISSUE TITLE
Only able to access 'Drive 1' and drive permanently running after loading program

### ISSUE THREAD

**Issue by robosnacks** [TomDDG/ZXPicoMD#9]
Hi, 
I wonder if you could offer some thoughts or advice?

I've built myself a unit a few days ago and it it doesn't quite seem to be behaving as I'd expected.

The 2 obvious problems I am seeing are:
- I only seem to be able to access 'Drive 1'. E.g. If I transfer 2 different  test carts to 'Drive 1' and 'Drive 2' respectively then `CAT 1`,  the displayed listing is as expected. However, if I then `CAT 2` I just get the 'Drive 1' listing again.
-  Loading programs from 'Drive 1' seems to work fine. However, once the program has loaded the 'Drive Motor On' LED remains on along with the buzzer and the SD Card access LED is pulsing permanently. The only way to stop it is to hit the RESET.

The provided diagnostic utilities all work successfully with one notable exception. The Martin Smith cart always hangs when it gets to `Reading a 'write' file, 160:1`

I have a Issue 2 IF1 according to the test utilities.

I've tested on my 2 Spectrums  (an Issue 3 and 4S) with the same result. I've cleaned the edge connectors and inspected my soldering work with no obvious problems found. I've also tried a few different SD cards, just in case.

In terms of hardware the 'big question mark' is around the IF1 I think. This is untested (until  now) and unfortunately I don't have another one nor a real microdrive to try with it - so I do wonder if there's a problem here, as I can't seem to find anything else obvious. I'd also note that I've never used a real microdrive before so it's perfectly possible I'm misunderstanding something...

Any thoughts/advice/ideas you could offer would be gratefully received. Thanks.

**Comment by liveboxandy** [comment]
Check Q3 and Q6 on the IF1. I had a similar problem when I built a replacement IF1. I changed both and it appeared to correct the issue.

**Comment by robosnacks** [comment]
I changed out both Q3 and Q6 without success, however, I then also changed Q1 and this seems to have done the trick! 

(Interestingly, on closer inspection, a ZTX213 was installed as Q1 - according to the IF1 schematic & service manual it should be a BC213L  - which is what I replaced it with. Perhaps the root of the problem? - I guess maybe a historical repair at some point in it's life...)

Really pleased to have it working properly, love it! Many thanks Tom and liveboxandy for pointing me in the right direction.
